common/db_store/error.py

```diff
@@ -0,0 +1,20 @@
+class DBError(Exception):
+
+    def __init__(self, message, rc=1, *args):
+        """Handle Exception in DB store."""
```

Comment:
check function doc format

Reply:
A one-liner docstring is sufficient to describe its behavior so I've added one-liner docstring.

Reply:
OK, if it is as per google format, for consistency purposes please check it once.